### PR TITLE
Dashboard header redesign

### PR DESCRIPTION
**Pull request by alexcherman**
dashboard-mobile
dashboard-desktop

### CODE REVIEW

File: universal-login-react/src/ui/UFlow/DashboardModal.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const DashboardModal = ({walletService, onClose}: DashboardModalProps) =>
   };
 
   return (
-    <div className={`udashboard ${useThemeClassFor()}`}>
+    <div className={`udashboard ${useClassFor('udashboard')}`}>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/UFlow/DashboardModal.tsx:42]: We should probably remove `udashobord` `unilogin-theme-.. unilogin-component-udashboard` is enough

Comment (alexcherman) [reply universal-login-react/src/ui/UFlow/DashboardModal.tsx:42]: Why should I remove ``unilogin-theme- ...``. We use this ``unilogin-theme -...+ unilogin-component -...`` combination in other components?

Comment (alexcherman) [reply universal-login-react/src/ui/UFlow/DashboardModal.tsx:42]: I'll remove ``dashboard`` class when redesign will be finished. Don't want to spend time renaming classes. I'll do it step by step and remove it when this class won't be needed.

Comment (JustynaBroniszewska) [reply universal-login-react/src/ui/UFlow/DashboardModal.tsx:42]: Ok 👍

**Pull request merged**